Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -174,6 +195,19 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
                 receiveCompletion: { completion in
                     switch completion {
                     case .failure(let error):
+                            do {
+                                _ = try LocalStorage.saveOfflineJob(
```

Comment:
Let's also add a log line above indicating that we're saving things in offline mode

Reply:
done